azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -283,6 +289,28 @@ internal class CallCompositeActivity : AppCompatActivity() {
         }
     }
 
+    private fun actionBarAccessibilitySetup() {
+        try {
+            findViewById<View>(R.id.action_bar)?.let { actionBarView ->
+                val viewGroup = actionBarView as ViewGroup
+                val buttonView = viewGroup.children.filter { it is AppCompatImageButton }.first()
+                buttonView.setOnKeyListener { _, keyCode, event ->
```

Comment:
this buttonView is a guess based on implementation of AppBarView, it could be the wrong button if they change implementation. Which would mean it could apply a keyListener to the wrong button.